docs/creating_component_code_review.md

```diff
@@ -47,8 +47,16 @@ bridge = Bridge(...)
 status = bridge.status()
 ```
 
-### 5. Limit platforms included in initial pull request
-Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. If your new component will have multiple platforms, try and limit your initial pull request to a single platform. Once the initial component is merged, you can submit additional PRs for the remaining platforms. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new feature in sooner. Pull requests containing large code dumps will not be a priority for review.
+### 5. Make your pull request as small as possible
+Keep a new integration to the minimum functionality needed for someone to get value out of the integration. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new integration/features in sooner. **Pull requests containing large code dumps will not be a priority for review and may be closed.**
+- Limit to a single platform 
+- Do not add features not needed to directly support the single platform (such as custom services)
+- Do not mix clean-ups and new features in a single pull request.
+- Do not solve several issues in a single pull request.
+- Do not submit pull requests that depend on other work which is still unmerged.
 
 ### 6. Event names
 Prefix component event names with the component name itself. For example, use `netatmo_person` instead of `person` for the `netatmo` component.
+
+### 7. Tests
+Strongly consider adding tests for your component to ensure it works correctly.
```

Comment:
Hmm, I can accept that statement but I don't much agree with it. Tests per se don't ensure anything and for an initial implementation, spending the same effort on manual testing will probably find more defects. Especially for a piece of software like an integration where most bugs are due to misunderstood/unknown properties of the external device. So, just a suggested rewording:

```suggestion
Strongly consider adding tests for your component to hinder future regressions.
```

Reply:
How about "minimize future regressions"?  Hinder sounded a little odd to me.

Reply:
That does sound better. I was struggling to find a word that did not promise to avoid regressions entirely.